codes/quantum/qubits/quantum_goppa.yml

```diff
@@ -0,0 +1,117 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: binary_quantum_goppa
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Binary Quantum Goppa Code' 
```

Comment:
I didn't change this from binary to Qubit quantum goppa because "binary" refers to usage of a binary field when constructing the classical Goppa code. There exist constructions of quantum goppa codes over fields of odd characteristic, but I did not describe that here. Let me know if you still think I should rename this.

Reply:
Thanks

Reply:
@mshingan This is a Galois-qudit code, no? In other words, a code for GF(q) qudits. I'm curious, what is the reason you set q=2^m? The m=1 case reduces to ordinary qubits.

Reply:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/8660306/146232733-17a162a7-f244-451b-8810-0722f1ffe8e9.png">